ts-apps/jl4-web/src/routes/+page.svelte

```diff
@@ -10,6 +10,7 @@
   import * as monaco from '@codingame/monaco-vscode-editor-api'
```

Comment:
Refactor jl4-web page.svelte: Make it more similar to vscode webview. 
Pass in a `makeLadderFlow` helper.